src/bundles/CaseStudy/components/SubmitConfirmation/SubmitConfirmation.js

```diff
@@ -12,28 +13,116 @@ class SubmitConfirmation extends React.Component {
         const {
             opportunityUrl,
             domain,
-            closingDate
+            closingDate,
+            briefTitle,
+            previewUrl,
+            profileUrl,
+            unassessedDomains,
+            assessedDomains
         } = this.props;
 
+        let hasUnassessed = !isEmpty(unassessedDomains) && Object
+            .keys(unassessedDomains)
+            .length > 0;
+        
+        let hasAssessed = !isEmpty(assessedDomains) && assessedDomains;
```

Comment:
This component is being targeted by two flows so assessedDomains may not be passed in, so I'm checking in one case where it is and it's empty and the other where we don't expect it